frontend/src/App.vue

```diff
@@ -0,0 +1,76 @@
+<script setup lang="ts">
+import { ref, computed, onMounted, onUnmounted } from 'vue';
+
+const offsetX = ref(0.0);
+const offsetY = ref(0.0);
+
+const windowWidth = ref(window.innerWidth);
+const windowHeight = ref(window.innerHeight);
+
+const viewBox = computed(() => {
+  return `${offsetX.value} ${offsetY.value} ${windowWidth.value} ${windowHeight.value}`;
+});
+
+const onWindowResized = () => {
+  windowWidth.value = window.innerWidth;
```

Comment:
You can add comments to the files in the file view.

This is just an example.

```suggestion
  windowWidth.value = window.innerWidth; // you can also suggest code changes by using the suggestion button
```

Reply:
Once a comment has been discussed to the satisfaction of everyone involved, the comment can be resolved.